src/store/modules/getters/wallet.js

```diff
@@ -0,0 +1,3 @@
+export const getAddress = state => state.address;
+export const getSigner = state => state.signer;
+export const getConnector = state => state.connector;
```

Comment:
When would we need to get the connector?

Reply:
We only use the connector in `connectWallet()` to call it's `openConnectWalletModal()`, not sure if we want to store it in the wallet store. Was thinking of declaring `connector` locally in the `/actions/wallet.js`, but we need `commit` param of the `connectWallet()` , so cannot declare `connector` outside of `connectWallet()`.
Should I remove the getter and mutation-type relative with `connector`?

Reply:
hmm maybe keep it for now and see if we use it later

Reply:
OK, then there is no pending issues for now.